docker-compose.yml

```diff
@@ -51,17 +51,34 @@ services:
     networks:
       - default
   qc-atlas:
-    image: planqk/atlas:v3.1.1
+    image: planqk/atlas:latest
     environment:
       POSTGRES_HOSTNAME: db
       POSTGRES_PORT: 5060
       POSTGRES_USER: planqk
       POSTGRES_PASSWORD: planqk
       POSTGRES_DB: planqk
+      WINERY_HOSTNAME: host.docker.internal
+      WINERY_PORT: 8091
     ports:
       - "6626:6626"
     networks:
       - default
+  winery:
+    image: opentosca/winery:latest
+    profiles: ["all", "winery"]
+    environment:
+      WINERY_PORT: 8091
+      WORKFLOWMODELER_PORT: 8091
+      TOPOLOGYMODELER_PORT: 8091
+      WINERY_HOSTNAME: host.docker.internal
```

Comment:
I'm not quite sure if this works correctly. Do we have a `.env`-file with a hostname? Maybe we should use this instead.

Reply:
I have added a .env file